app/src/main/kotlin/dev/patrickgold/florisboard/app/settings/media/MediaScreen.kt

```diff
@@ -41,6 +50,11 @@ fun MediaScreen() = FlorisScreen {
     previewFieldVisible = true
     iconSpaceReserved = true
 
+    val prefs by florisPreferenceModel()
+
+    val shouldDelete = remember { mutableStateOf<ShouldDelete?>(null) }
```

Comment:
COuld be improved by using the Kotlin delegates (will require 2 additional imports but Android Studio suggests them immediately)

```suggestion
    var shouldDelete by remember { mutableStateOf<ShouldDelete?>(null) }
```